### ISSUE TITLE
amagama update?

### ISSUE THREAD

**Issue by mzeidhassan** [translate/amagama#3220]
Hi @friedelwolff ,

I hope all is well with you. I see that you are making changes to amagama. Does this mean that the project is live again and you plan to update it, maybe to Python 3?

I thought I should ask you.

Thanks

**Comment by friedelwolff** [comment]
Yes, I made a number of updates. I hope to update the database soon, I have done most of the hard work a while ago.

I believe I finished the upgrade to Python 3 - there was very little that had to be changed, if I remember correctly. Feel free to try it out and say how it goes. The tests are already running for Python3 versions:
https://travis-ci.org/translate/amagama/

**Comment by marcmittag** [comment]
Hi @friedelwolff ,
is there any news on this? Does Amagama run with the latest python?
best
Marc
